src/main/java/nl/tudelft/cse1110/andy/result/Result.java

```diff
@@ -14,8 +14,9 @@
     private final String genericFailure;
     private final double timeInSeconds;
     private final GradeWeight weights;
+    private final String externalProcessOutput;
```

Comment:
We do not care about the external process output, unless it's an error, in which case we just set is as "genericFailure"